eng/Helix.SingleTarget.targets

```diff
@@ -1,13 +1,56 @@
 <Project>
+
+  <PropertyGroup>
+    <!-- Azurite requires Node -->
+    <HelixRequiresNode Condition="'$(HelixRequiresAzurite)' == 'true'">true</HelixRequiresNode>
+  </PropertyGroup>
+
+  <!-- Skip installation on musl libc systems (node is not supported) -->
+  <PropertyGroup Condition="'$(IsHelixMuslLibc)' == 'true'">
+    <HelixRequiresAzurite>false</HelixRequiresAzurite>
+    <HelixRequiresNode>false</HelixRequiresNode>
+  </PropertyGroup>
+
   <ItemGroup>
-    <!-- Set TEST_DOTNET_ROOT to same as DOTNET_ROOT (which is provided by the dotnet cli correlation payload) -->
+    <!-- Helper to dynamically use a semicolon for path separation at execution time. -->
+    <HelixWorkItemPreCommand Include="SET /p semicolon=%3C%HELIX_CORRELATION_PAYLOAD%\semicolon.txt"
+                             Condition="'$(IsHelixPosixShell)' != 'true'" />
+    <!-- Set TEST_DOTNET_ROOT to same as DOTNET_ROOT (which is provided by ) -->
     <HelixWorkItemPreCommand Include="export TEST_DOTNET_ROOT=${DOTNET_ROOT}"
                              Condition="'$(IsHelixPosixShell)' == 'true'" />
     <HelixWorkItemPreCommand Include="SET TEST_DOTNET_ROOT=%DOTNET_ROOT%"
                              Condition="'$(IsHelixPosixShell)' != 'true'" />
   </ItemGroup>
 
-  <!-- TODO: Add Helix work item PreCommands to install Node and Azurite -->
+  <!-- Configure NodeJs and Npm -->
+  <ItemGroup Condition="'$(HelixRequiresNode)' == 'true'">
```

Comment:
I might be able to put each of these blocks into cmd/sh scripts that are sourced from the dynamically generated script in order to simplify this a bit. It would also remove the semicolon workaround for prepending new paths to PATH on Windows.